.github/workflows/run_mmperf.yml

```diff
@@ -51,10 +52,17 @@ jobs:
           # https://github.com/nektos/act where there will be more.
           git log --oneline --graph --max-count=3
           ./build_tools/github_actions/configure_ci.py
+      - name: "Checkout out SHARK-Runtime"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e  # v2
+        with:
+          repository: nod-ai/SHARK-Runtime
+          ref: main
+          path: ${{ github.workspace }}/SHARK-Runtime
       - name: "Calculating version info"
         id: iree
         run: |
-          export IREE_SHA=$(git rev-parse --short HEAD)
+          cd ${{ github.workspace }}/SHARK-Runtime
+          export IREE_SHA=`git rev-parse --short=12 HEAD`
```

Comment:
FYI, there's no need to `export` here, since there aren't any child processes that could be using this.

Reply:
Thanks, I removed the export.